api/swagger.yaml

```diff
@@ -8331,6 +8331,15 @@ paths:
           description: "BuildKit output configuration"
           type: "string"
           default: ""
+        - name: "version"
+          in: "query"
+          type: "string"
+          default: "1"
+          description: |
+            Version of the builder backend to use.
+
+            - `1` is the first generation classic builder in the Docker daemon (default)
```

Comment:
```suggestion
            - `1` is the first generation classic (deprecated) builder in the Docker daemon (default)
```

Reply:
Do we want to call it deprecated if it's default?

Reply:
If you try and build with the classic builder you get

```
$ DOCKER_BUILDKIT=0 docker build ....
DEPRECATED: The legacy builder is deprecated and will be removed in a future release.
            BuildKit is currently disabled; enable it by removing the DOCKER_BUILDKIT=0
            environment-variable.
```

Reply:
But yeah, the default being the deprecated one isn't really ideal I'll give you that

Reply:
I think we're not printing that message for native Windows (as there's no buildkit support yet), so it's a bit of a "in between state".

I'm personally fine with leaving the `(deprecated)` for a follow-up, but also ok with doing it now 🤷 

We could also add a separate block below the list of options which would give a bit more room to outline;

```
Deprecated: .....
```

Reply:
Added `(deprecated`).